sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -40,8 +40,10 @@ public SessionHandlerOptions(Func<ExceptionReceivedEventArgs, Task> exceptionRec
         /// When errors are received calls will automatically be retried, so this is informational. </summary>
         public Func<ExceptionReceivedEventArgs, Task> ExceptionReceivedHandler { get; }
 
-        /// <summary>Gets or sets the duration for which the session lock will be renewed automatically.</summary>
-        /// <value>The duration for which the session renew its state.</value>
+        /// <summary>Gets or sets the duration for which the session lock will be renewed automatically. If a session lock is going to expire and 
+        /// there are still active messages in the session or operations are not yet complete in the current session, this value is the max duration 
+        /// for the session lock to be automatically renewed after the current session lock expires. </summary>
+        /// <value>The duration for which the session renew its state. Default is 5 min.</value>
         public TimeSpan MaxAutoRenewDuration
```

Comment:
Could you add a `<remarks>  </remarks>` and mention that if renewal fails, they will receive an exception in the exceptionHandler